kubernetes/client/clientset/clientset.go

```diff
@@ -1,96 +1,23 @@
 package clientset
 
 import (
-	composev1beta1 "github.com/docker/cli/kubernetes/client/clientset/typed/compose/v1beta1"
-	composev1beta2 "github.com/docker/cli/kubernetes/client/clientset/typed/compose/v1beta2"
-	"github.com/golang/glog"
-	"k8s.io/client-go/discovery"
-	"k8s.io/client-go/rest"
-	"k8s.io/client-go/util/flowcontrol"
+	api "github.com/docker/compose-on-kubernetes/api/client/clientset"
```

Comment:
WIth these now being a single import, we should probably remove the brackets, and change them to

```suggestion
import api "github.com/docker/compose-on-kubernetes/api/client/clientset"
```